ui/ui-components/components/import/List.tsx

```diff
@@ -171,16 +168,6 @@ export default function ImportList(props) {
                     {_import.importedAt
                       ? formatTimestamp(_import.importedAt)
                       : "pending"}
-                    <br /> Processed:{" "}
-                    {_import.processedAt
-                      ? formatTimestamp(_import.processedAt)
-                      : "pending"}
-                  </td>
-                  <td>
-                    <ImportRecordPropertiesDiff _import={_import} />
-                  </td>
-                  <td>
-                    <ImportGroupsDiff _import={_import} groups={groups} />
```

Comment:
Hmm... I wonder if writing less data is worth losing the visibility/traceability of changes 🤔

Reply:
That's the crux of the whole PR - @bleonard and I came to the conclusion that this data is mostly redundant.  We keep change timestamps on each RecordProperty now so you can see when values were checked & changed.  GroupMembers have similar timestamps.  What's lost is data for the negative case (eg: I used to be in this group, and now I'm not).  If we do want to bring that data back at some point we can, but I believe it belongs in some other format, like a "RecordSnapshots" table - something that doesn't live in the critical import/export path whose writes would slow things down.

We do want to continue to keep this information on the export, because that's actually used to compute the diffs about what we send on the way out, and is what users care more about.